src/components/clusterWizard/RoleDropdown.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Host, ClusterUpdateParams } from '../../api/types';
+import { SimpleDropdown } from '../ui/SimpleDropdown';
+import { patchCluster } from '../../api/clusters';
+import { HostRolesType, HOST_ROLES } from '../../config/constants';
+
+type RoleDropdownProps = {
+  role: Host['role'];
+  host: Host;
+};
+
+export const RoleDropdown: React.FC<RoleDropdownProps> = ({ role, host }) => {
+  const [isDisabled, setDisabled] = React.useState(false);
+
+  const setRole = async (role?: string) => {
+    const params: ClusterUpdateParams = {};
+    setDisabled(true);
+    params.hostsRoles = [
+      {
+        id: host.id,
+        role: role as HostRolesType,
+      },
+    ];
+    // TODO(mlibra): store updatedCluster (infra WIP) or initiate reload
+    // const updatedCluster =
```

Comment:
Will be stored to redux and hosts will be retrieved from Cluster (not as recently via specific request). WIll be Follow-up for https://github.com/openshift-metal3/facet/pull/71